components/mixins/profile/profile-person-details-v2.schema.json

```diff
@@ -0,0 +1,287 @@
+              "type": "integer",
+              "description": "The year a person was born including the century, for example, 1983.  This field should be used when only the person's age is known, not the full birth date.",
+              "minimum": 1,
+              "maximum": 32767
+            }
+          }
+        }
+      }
+    },
+    "person-gender": {
+      "properties": {
+        "xdm:person": {
+          "type": "object",
+          "properties": {
+            "xdm:gender": {
+              "title": "Gender",
+              "type": "string",
+              "enum": ["male", "female", "not_specified", "non_specific"],
+              "meta:enum": {
+                "male": "Male",
+                "female": "Female",
+                "not_specified": "Not Specified",
+                "non_specific": "Non-specific"
+              },
+              "description": "Gender identity of the person.\n",
+              "default": "not_specified"
+            }
+          }
+        }
+      }
+    },
+    "person-maritalStatus": {
+      "properties": {
+        "xdm:person": {
+          "type": "object",
+          "properties": {
+            "xdm:maritalStatus": {
+              "title": "Marital Status",
+              "type": "string",
+              "enum": [
+                "married",
+                "single",
+                "divorced",
+                "widowed",
+                "not_specified"
+              ],
+              "meta:enum": {
+                "married": "Married",
+                "single": "Single",
+                "divorced": "Divorced",
+                "widowed": "Widowed",
+                "not_specified": "Not Specified"
+              },
+              "description": "Describes a person's relationship with a significant other.",
+              "default": "not_specified"
+            }
+          }
+        }
+      }
+    },
+    "person-nationality": {
+      "properties": {
+        "xdm:person": {
+          "type": "object",
+          "properties": {
+            "xdm:nationality": {
+              "title": "Nationality",
+              "type": "string",
+              "pattern": "^[A-Z]{2}$",
+              "description": "The legal relationship between a person and their state represented using the ISO 3166-1 Alpha-2 code."
+            }
+          }
+        }
+      }
+    },
+    "person-taxId": {
+      "properties": {
+        "xdm:person": {
+          "type": "object",
+          "properties": {
+            "xdm:taxId": {
+              "title": "Tax ID",
+              "type": "string",
+              "description": "The Tax / Fiscal ID of the person, e.g. the TIN in the US or the CIF/NIF in Spain."
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    { "$ref": "#/definitions/person" },
+    { "$ref": "#/definitions/person-birthDate" },
```

Comment:
The definition on line 271 already brings in the xdm:person fields with all of its sub-fields. You don't need all these other ref's in the allOf. You only need #/definitions/person

Reply:
Done